SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -52,6 +53,16 @@ void FSpatialGDKEditorModule::RegisterSettings()
 		{
 			RuntimeSettingsSection->OnModified().BindRaw(this, &FSpatialGDKEditorModule::HandleRuntimeSettingsSaved);
 		}
+
+		ISettingsSectionPtr CloudLauncherSettingsSection = SettingsModule->RegisterSettings("Project", "SpatialGDKEditor", "Cloud Settings",
+			LOCTEXT("SpatialCloudLauncherGeneralSettingsName", "Cloud Settings"),
```

Comment:
I wonder if we should call this `cloud deployment settings` or just `cloud settings`?

Reply:
IMO `Cloud Deployment Settings`  better reflects what the settings are about at this point.

Reply:
Changed it to `Cloud Deployment Settings` 👍